app/src/main/java/com/duckduckgo/app/browser/tabs/adapter/FragmentStateAdapter.java

```diff
@@ -107,7 +108,6 @@ public abstract class FragmentStateAdapter extends RecyclerView.Adapter<Fragment
     @SuppressWarnings("WeakerAccess") // to avoid creation of a synthetic accessor
     final LongSparseArray<Fragment> mFragments = new LongSparseArray<>();
 
-    private final LongSparseArray<Fragment.SavedState> mSavedStates = new LongSparseArray<>();
```

Comment:
We don't use stored fragment state itself, we just need to know what fragments are stale in order to remove them when recreating an adapter, so we can remove this as it's only taking up space in a saved bundle.